hieradata/site/ls/role/rke.yaml

```diff
@@ -1,3 +1,2 @@
 ---
 profile::core::docker::version: "24.0.9"
```

Comment:
I think this entire file can go as `profile::core::docker::version: "24.0.9"` is set in `profile::core::docker::version`. I suspect we can bump the docker version in `site/profile/data/common.yaml` and remove it from `site/<site>.yaml` completely (in another PR).